app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self):
+        print(f"{self.nickname} is eating {self._favourite_dish}")
+
+        super().get_rating()

```

Comment:
You don't need to use super() in here, it`s will work without it

Reply:
All changes have been done. Thank you for your review!